Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -278,14 +277,8 @@ class MessageQueue {
       }
     }
 
-    /* $FlowFixMe(>=0.38.0 site=react_native_fb,react_native_oss) - Flow error
-     * detected during the deployment of v0.38.0. To see the error, remove this
-     * comment and run flow */
-    this._callbacks[cbID & ~1] = null;
-    /* $FlowFixMe(>=0.38.0 site=react_native_fb,react_native_oss) - Flow error
-     * detected during the deployment of v0.38.0. To see the error, remove this
-     * comment and run flow */
-    this._callbacks[cbID |  1] = null;
+    this._successCallbacks[callID] = this._failureCallbacks[callID] = null;
+
     // $FlowIssue(>=0.35.0) #14551610
     callback.apply(null, args);
```

Comment:
One way to please flow here would be to take the null check from line 275 and move it outside of the `else` clause.